### DISCUSSION TITLE
Rename keycode functions

### DISCUSSION THREAD

**Discussion by NicoHood** [NicoHood/HID#278] [Ideas]
I get lot of bug reports about people trying to use keycodes directly, but they are casting them wrong. The idea is to rename functions like `inline size_t write(KeyboardKeycode k)` to `inline size_t writeKeycode(KeyboardKeycode k)`.
https://github.com/NicoHood/HID/blob/master/src/HID-APIs/KeyboardAPI.h#L38-L44

We could deprecate the old functions to start a smooth transition. What do you think?

CC @SukkoPera

**Comment by SukkoPera** [comment]
(I'm currently thinking on this matter, will get back to you ASAP.)
